{
  "id": "CVE-2020-28189",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2020-12-24T15:15:13.403",
  "lastModified": "2020-12-24T15:15:13.403",
  "vulnStatus": "Rejected",
  "descriptions": [
    {
      "lang": "en",
      "value": "** REJECT ** DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: CVE-2020-29189. Reason: This candidate is a reservation duplicate of CVE-2020-29189. Notes: All CVE users should reference CVE-2020-29189 instead of this candidate. All references and descriptions in this candidate have been removed to prevent accidental usage."
    }
  ],
  "metrics": {},
  "references": []
}